### DISCUSSION TITLE
Persisting Settings (How to save User Preferences)

### DISCUSSION THREAD

**Discussion by DrastiGohel** [serenity-is/Serenity#6601] [Help]
Hi all,

Can someone help out with how to set user preferences in Serenity Version 6.3.2?

**Comment by DrastiGohel** [comment]
Hi all, 

I figured out the way to solve this.

Thanks

**Reply by edwardch** [reply]
Hey @DrastiGohel, 

Please consider posting the link or code to the solution, even if it is to your own question.

Then if you mark it as the answer the rest of the community can learn from it, too 😄 

Thank you,
Edward
